tests/build.rs

```diff
@@ -80,7 +95,21 @@ fn it_fails_with_multiple_webpack_configs() {
         type = "Webpack"
     "#};
 
-    build_fails_with(fixture, "multiple webpack configurations is not supported.");
+    build_fails_with(fixture, "Multiple webpack configurations are not supported. You can specify a different path for your webpack configuration file in wrangler.toml with the `webpack_config` field");
+    cleanup(fixture);
+}
+
+#[test]
+fn it_fails_with_multiple_specify_webpack_configs() {
+    let fixture = "webpack_multiple_specify_config";
+    create_temporary_copy(fixture);
+
+    settings! {fixture, r#"
+        type = "Webpack"
+        webpack_config = "webpack.worker.js"
+    "#};
+
+    build_fails_with(fixture, "Multiple webpack configurations are not supported. You can specify a different path for your webpack configuration file in wrangler.toml with the `webpack_config` field");
```

Comment:
long term... testing on the full error message might make this test a little brittle. i think it's totally fine to keep now, but if we find the error message changes slightly a lot over time... we may just want to test that the error message contains some keywords that we know won't change and are enough to identify it :)